### PR TITLE
Replace <+> with <>

### PR DESCRIPTION
**Pull request by johnli360**
### Description

Replaces <+> usage with <> as discussed in https://github.com/xmonad/xmonad/issues/132#issuecomment-886218490

### Checklist

  - [x] I've read [CONTRIBUTING.md](https://github.com/xmonad/xmonad/blob/master/CONTRIBUTING.md)

  - [x] I've considered how to best test these changes (property, unit,
        manually, ...) and concluded: I ran `cabal build` using ghc 8.4.4, and 9.2.2

  - [x] I updated the `CHANGES.md` file

### CODE REVIEW

File: XMonad/Doc/Extending.hs [hunk 1]
```diff
@@ -347,7 +347,7 @@ Since @mergeKeylist@ now has exactly the right type signature, we can
 just set that as the keys.
 
 If you like operators, 'Data.Monoid.<>' (or xmonad's alias for it,
-'XMonad.ManageHook.<+>') does exactly the same as the explicit usage of
+'XMonad.ManageHook.<>') does exactly the same as the explicit usage of
```
Comment (slotThe) [inline XMonad/Doc/Extending.hs:350]: This should stay as `<+>`

File: XMonad/Doc/Extending.hs [hunk 2]
```diff
@@ -617,7 +617,7 @@ RealPlayer on the float layer, the second one will put the xpdf
 windows in the workspace named \"doc\", with 'XMonad.ManageHook.doF'
 and 'XMonad.StackSet.shift' functions, and the third one will put all
 firefox windows on the workspace called "web". Then we use the
-'XMonad.ManageHook.<+>' combinator to compose @myManageHook@ with the
+'XMonad.ManageHook.<>' combinator to compose @myManageHook@ with the
```
Comment (slotThe) [inline XMonad/Doc/Extending.hs:620]: This should either say `<>` (without the qualification) or stay as `<+>`

File: XMonad/Doc/Extending.hs [hunk 3]
```diff
@@ -667,10 +667,10 @@ Then we create our own 'XMonad.Config.manageHook':
 
 >    myManageHook = resource =? "realplay.bin" --> doFloat
 
-We can now use the 'XMonad.ManageHook.<+>' combinator to add our
+We can now use the 'XMonad.ManageHook.<>' combinator to add our
```
Comment (slotThe) [inline XMonad/Doc/Extending.hs:670]: This should either say `<>` (without the qualification) or stay as `<+>`

File: XMonad/Util/PureX.hs [hunk 4]
```diff
@@ -71,7 +71,7 @@ import Control.Monad.Reader
 -- actions as @XLike m => m Any@ or @PureX Any@ values, where the encapsulated
 -- @Any@ value encodes whether or not a refresh is needed to properly institute
 -- changes. These values can then be combined monoidally (i.e. with '<>' AKA
--- '<+>') or with operators such as '<*', '*>', '<?' and '&>' to build seamless
+-- '<>') or with operators such as '<*', '*>', '<?' and '&>' to build seamless
```
Comment (slotThe) [inline XMonad/Util/PureX.hs:74]: This should stay as `<+>`

**Review by slotThe — COMMENTED** [review]
This look nice, thanks!  I think, as a general strategy, it's wise to not deprecate `<+>` (I would wager almost all users have this in their config in some form, making a deprecation very costly for little benefit).  However, having our internal use cleaned up seems like the right thing to do

File: XMonad/Doc/Extending.hs (hunk 1 above)
Comment (johnli360) [reply XMonad/Doc/Extending.hs]: Oh yeah, sorry about that

**Comment by slotThe** [comment]
Thank you!

**Pull request merged**